lighthouse-core/report/report-generator.js

```diff
@@ -73,7 +73,7 @@ class ReportGenerator {
       return category.audits.map(catAudit => {
         const audit = lhr.audits[catAudit.id];
         return [category.name, audit.name, audit.description, audit.scoreDisplayMode, audit.score]
-          .map(value => value.toString())
+          .map(value => value === null ? '0' : value.toString())
```

Comment:
i dont know what the CSV consumers want but i feel like getting 0 is a little weird. empty string seems slightly better.

Reply:
csv-validator requires that all values in the column are numbers, I have no preference here we can ditch csv-validator if we want to deviate?  

cc @nourikhalass in case you have a preference for how `null` scores show up in CSV

Reply:
I agree with @paulirish that 0 is a bit weird so empty string would be better. But as @patrickhulce says: csv-validator does not play along with that.  Maybe its better to use -1 or omit the result entirely if the score is null.

Reply:
I'll go with -1 for now since you're our primary CSV consumer, PR welcome in the future if you think of a better canonical CSV-way to represent this :)

Reply:
I think that's fine since the validation library leaves a lot to be desired. Maybe in the future I'll look at a better / more robust way of validating the CSV . So I think you'll be OK if you don't validate that scores are numeric.

Also thanks for including me in the loop!